### PR TITLE
fix(glam): add separate import revisions step

### PR DESCRIPTION
**Pull request by edugfilho**
## Description

Fixes https://github.com/mozilla/glam/issues/2989 by adding a separate step to import build revisions into GLAM's database.

## Related Tickets & Documents
* https://github.com/mozilla/glam/issues/2989

### CODE REVIEW

File: dags/glam_glean_imports.py [hunk 1]
```diff
@@ -131,16 +131,16 @@
...
 base_docker_args = ["/venv/bin/python", "manage.py"]
 
 for env in ["Dev", "Prod"]:
     glam_import_image = default_glam_import_image
     if env == "Dev":  # noqa 114
-        glam_import_image = "gcr.io/moz-fx-dataops-images-global/gcp-pipelines/glam/glam-production/glam:2023.07.1-43"
+        glam_import_image = "gcr.io/moz-fx-dataops-images-global/gcp-pipelines/glam/glam-production/glam:2024.10.0-58"
     elif env == "Prod":
-        glam_import_image = "gcr.io/moz-fx-dataops-images-global/gcp-pipelines/glam/glam-production/glam:2023.07.1-43"
+        glam_import_image = "gcr.io/moz-fx-dataops-images-global/gcp-pipelines/glam/glam-production/glam:2024.10.0-58"
```
Comment (BenWu) [inline dags/glam_glean_imports.py:143]: Why is this needed when all the images are the same?

**Review by BenWu — APPROVED** [review]

File: dags/glam_glean_imports.py (hunk 1 above)
Comment (edugfilho) [reply dags/glam_glean_imports.py:143]: GLAM has different psql dbs for dev and prod, so this would allow for testing of more complicated things. That said, I don't think I've ever seen it being used.
